core/dbt/parser/schema_yaml_readers.py

```diff
@@ -499,7 +499,6 @@ def _get_measures(self, unparsed_measures: List[UnparsedMeasure]) -> List[Measur
                     name=unparsed.name,
                     agg=AggregationType(unparsed.agg),
                     description=unparsed.description,
-                    create_metric=unparsed.create_metric,
```

Comment:
Not passing it here is fine because the [Measure class](https://github.com/dbt-labs/dbt-core/blob/b2449262f3cd1a99447645a6c1572195b10c2f49/core/dbt/contracts/graph/semantic_models.py#L135) that exists on the [SemanticModel](https://github.com/dbt-labs/dbt-core/blob/b2449262f3cd1a99447645a6c1572195b10c2f49/core/dbt/contracts/graph/nodes.py#L1473) node in the graph [defaults the value to false](https://github.com/dbt-labs/dbt-core/blob/b2449262f3cd1a99447645a6c1572195b10c2f49/core/dbt/contracts/graph/semantic_models.py#L139)